tests/Oryx.Tests.Common/EndToEndTestHelper.cs

```diff
@@ -264,6 +266,40 @@ public static class EndToEndTestHelper
             }
         }
 
+        public static Task RunPackAndAssertAppAsync(
+            ITestOutputHelper output,
+            string appName,
+            DockerVolume appVolume,
+            string appImageName,
```

Comment:
Is this the name of the final runtime image that gets built as part of this build pack process?

Reply:
To prevent tests running in parallel trumping on same names, should we generate unique names for these images?

Reply:
Think about untagging these images too as otherwise they would consume disk space in view of you might be writing more tests in this area. You could follow a naming pattern (like we do for container names) like `oryxtests_*` for these images and can untag them in existing `dockerCleanup.sh` script